array-manipulation-2/src/drop.js

```diff
@@ -0,0 +1,15 @@
+/* exported drop */
+// create empty array and set a condition for task with empty array (array.length = 0);
+// run for loop but make i = count (instead of i=0), and push array[i] to result array
+function drop(array, count) {
+  var result = [];
+  if (array.length === 0) {
```

Comment:
This check isn't actually necessary since the loop on line 9 won't execute if array.length is 0.